.changeset/nervous-spoons-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@shopware-ag/meteor-component-library": minor
+---
+
+Implement toasts
```

Comment:
I write general foundings from my UI testing here:

- Adding several toasts in the middle shows a animation very shortly which looks a bit weird. If we can fix this this would be cool
- Hover effect for closing "X" is missing (if wanted by design)
- A title attribute for long text would be good so that it could be readable on hover (https://www.w3schools.com/tags/att_global_title.asp#:~:text=HTML%20title%20Attribute&text=The%20title%20attribute%20specifies%20extra,it%20is%20not%20necessarily%20useful).)

Reply:
Thanks to your help I was able to resolve all of this 😊